README.md

```diff
@@ -1,104 +1,215 @@
-# ntStat
+ntStat is a toolkit that uses Bloom filters to track both *k*-mer count and depth information for use in downstream applications. ntStat also models the *k*-mer count histogram using a mixture model, and infers valuable insights about the genome, sequencing data, and individual *k*-mers, *de novo*.
+
+**Contents**
+- [Installation](#installation)
+  - [From source](#from-source)
+- [Dependencies](#dependencies)
+  - [General dependencies](#general-dependencies)
+  - [C++ libraries](#c-libraries)
+  - [Python packages](#python-packages)
+- [Modules](#modules)
+  - [`count`](#count)
+  - [`filter`](#filter)
+  - [`query`](#query)
+  - [`hist`](#hist)
+- [Credits](#credits)
+- [License](#license)
+
+# Installation
+
```

Comment:
TBD.  Here you'd include the conda install command. In the mean time, perhaps you could have basic instructions for cloning/building the repo?